google/cloud/pubsub/subscription_admin_client.h

```diff
@@ -104,6 +104,24 @@ class SubscriptionAdminClient {
     return connection_->GetSubscription({std::move(subscription)});
   }
 
+  /**
+   * Update an existing subscription in Cloud Pub/Sub.
+   *
+   * @par Idempotency
+   * This is not an idempotent operation and therefore it is never retried.
+   *
+   * @par Example
+   * @snippet samples.cc update-subscription
+   *
+   * @param subscription the name for the subscription
+   * @param builder any additional configuration for the subscription
+   */
+  StatusOr<google::pubsub::v1::Subscription> UpdateSubscription(
+      Subscription const& subscription, SubscriptionMutationBuilder builder) {
```

Comment:
Do I presume correctly there is no good reason to default the `SubscriptionMutationBuilder` ala `CreateSubscription()`?

Reply:
Correct, with `CreateSubscription()` you must provide at least the topic name and subscription name, and you can get away with providing just that.  With `UpdateSubscription()` you must provide the subscription name **and** you need to change something, a default builder says "change nothing".